topics/emacs/index.md

```diff
@@ -0,0 +1,12 @@
+---
+topic: emacs
```

Comment:
How about adding `"related": vim, emacs-lisp` and `"released": 1976`?

Reply:
I can add related tags, but I'm hesitant to add `released:` (and `author:`) data since there's not clear consensus.

What are your thoughts?

Depending on your answer, I'll push an updated commit tonight when I get home.

Reply:
Oh sure, if the release date and author are contentious, I'm fine omitting them! Thanks.

Reply:
Done 😄